shared/constants/team-building.tsx

```diff
@@ -172,15 +178,3 @@ const interestingPersonToUser = (person: RPCTypes.InterestingPerson): Types.User
     username: username,
   }
 }
-
-export {
```

Comment:
? Is this a new pattern we're following now?

Reply:
I think so. It just adds a bunch of lines for not much gain

Reply:
Great I'm all for it.